dm/master/server.go

```diff
@@ -25,6 +25,7 @@ import (
 	"sync"
 	"time"
 
+	"github.com/labstack/echo/v4"
```

Comment:
will this library be used for demonstrating only in this PR, or used from now on?

Reply:
it will be used from now